x/bank/CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* [#18956](https://github.com/cosmos/cosmos-sdk/pull/18956) Introduced a new `DenomOwnersByQuery` query method
```

Comment:
I believe this changelog should be under features instead of improvements.
Same goes for main changelog too.